src/app/routes.js

```diff
@@ -44,34 +45,5 @@ const appRoutes = [
   },
 ]
 
+createConfig(appRoutes)
```

Comment:
```
import routes from 'routes'
```

inside routerConfig

Reply:
how will u then use it as independent module?